### PR TITLE
kucoinfutures websockets

### PR DESCRIPTION
**Pull request by samgermain**

### CODE REVIEW

File: js/pro/kucoinfutures.js [hunk 1]
```diff
@@ -0,0 +1,968 @@
...
+        const subscription = {
+            'id': nonce.toString (),
+            'symbol': symbol,
+            'topic': topic,
+            'messageHash': messageHash,
+            'method': method,
+        };
+        const request = this.extend (subscribe, params);
+        const subscriptionHash = topic;
+        return await this.watch (url, messageHash, request, subscriptionHash, subscription);
```
Comment (samgermain) [inline js/pro/kucoinfutures.js:138]: @pcriadoperez Hey I thought you might be able to help here, nothing is being returned from this method with `watchTicker`, when you turn on `--verbose` though, you get the response from the exchange, but for some reason the parsed response isn't being returned

**Comment by cheunhong** [comment]
@samgermain I have made a PR for some fix on this branch

**Comment by cheunhong** [comment]
May I know what else is pending on this branch for it be merge ready?

**Comment by carlosmiei** [comment]
> May I know what else is pending on this branch for it be merge ready?

@cheunhong  I will review this soon, so that we can merge it asap

**Comment by cheunhong** [comment]
@carlosmiei thanks

**Comment by carlosmiei** [comment]
> @carlosmiei thanks



> @carlosmiei thanks

@cheunhong btw you said you added some fixes to this branch, should I review yours then?

**Comment by cheunhong** [comment]
@carlosmiei sure, PR is on @samgermain's fork

**Comment by carlosmiei** [comment]
> @carlosmiei sure, but the PR is on @samgermain's fork

@cheunhong can't you open it here? would be easier

**Comment by cheunhong** [comment]
@carlosmiei ok no problem, this is the new PR https://github.com/ccxt/ccxt/pull/16422

**Comment by samgermain** [comment]
reopened in #16422

**Pull request closed without merging**